download-funcs.sh

```diff
@@ -71,12 +71,13 @@ apply_patch() {
 }
 
 download_glibc () {
-   if [ -d glibc ]; then
-      cd glibc && git pull && cd ..
-      return 0
+   if [ ! -d glibc ]; then
+      git clone git://sourceware.org/git/glibc.git
    fi
-   git clone git://sourceware.org/git/glibc.git &&
+
    cd glibc &&
+   git reset --hard &&
+   git checkout $GLIBC_TAG &&
```

Comment:
Not 100% happy with this solution but it works alright. If you have a better solution I'd be happy to change this